runtime/lua/vscode-neovim/api.lua

```diff
@@ -291,14 +291,6 @@ do
   local op_func_id = 0
 
   ---@see map-operator
-  --- vim.keymap.set(
```

Comment:
I think this usage is good to have somewhere, maybe the readme?

Reply:
I don't know what to write.. This function is simply encapsulated, and the details are in the vim document.
Maybe more examples can be added later.